sleap_roots/angle.py

```diff
@@ -41,18 +41,36 @@ def get_node_ind(pts: np.ndarray, proximal: bool = True) -> np.ndarray:
         return np.nan
 
     if proximal:
```

Comment:
sorry I tried to vectorize this but looks like I did it incorrectly. If it was better before you can change it back!

Reply:
No problem! The current version also works and uses vectors instead of a for loop.